packages/google_sign_in/google_sign_in_android/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 6.0.0
+
+* Deprecate `clientId` and support `serverClientId` instead.
+  Historically `clientId` was interpreted as `serverClientId`, but only on Android. On
+  other platforms it was interpreted as the OAuth `clientId` of the app. For backwards-compatibility
+  `clientId` will still be used as a server client ID if `serverClientId` is not provided.
+* **BREAKING CHANGES**:
+  * Add `serverClientId` parameter to `IDelegate.init`.
```

Comment:
I didn't remember we usually consider platform side breaking change as plugin breaking change. Is it still the case?

cc @stuartmorgan

Reply:
Done